pallets/loans/src/lib.rs

```diff
@@ -1010,7 +990,7 @@ impl<T: Config> Pallet<T> {
     pub fn borrow_balance_stored_with_snapshot(
         currency_id: &CurrencyId,
         snapshot: BorrowSnapshot,
-    ) -> result::Result<Balance, Error<T>> {
+    ) -> result::Result<Balance, DispatchError> {
```

Comment:
I forgot why do we need to type `result::Result` instead of `Result`

Reply:
Because we can not use `std::Result` in wasm runtime. 

`sp_std::result` is provided by substrate.

Reply:
then we can import sp_std::result::Result maybe

Reply:
That makes sense.

Reply:
Modified